submissions/saudatSalawudeen/html-forms/contact-form.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Contact Us</title>
+</head>
+<body>
+	<!-- header -->
+	<header>
+		<div id="head">
+			<h1>ZARIS</h1>
+			<p><em>home of sexy bodice and underwears!</em></p>
+		</div>
+		<nav>
+			<p>
+				<a href="#">Home</a> |
+				<a href="#">About-Us</a> |
+				<a href="contact-form.html">Contact Form</a> |
+				<a href="product.html">Product</a>
+			</p>
+			<form>
+				<p>
+					<label for=""></label>
+					<input type="search" name="" id="" placeholder="product search">
+				</p>
+			</form>
+		</nav>
+	</header>
+	<hr/>
+	
+	<header>
+		<h2><u>Let Us Connect With You!</u></h2>
+	</header>
+	
+	<!-- form content -->
+	<main>
+		<form action="#" method="post" id="form">
+			<div class="container" id="">
+				<div class="">
+					<label for="fullname">Full Name</label>
+					<input type="text" name="fullname" id="fullname" placeholder="Enter your first and last name" required />
+				</div><br/>
+				
+				<div class="form-box">
+					<label for="email">Email</label>
+					<input type="email" name="email" id="email" placeholder="Enter your email" required />
+				</div><br/>
+				
+				<div class="form-box">
+					<label for="subject">Subject</label>
+					<input type="text" name="subject" id="subject" maxlength="20"  />
+				</div><br/>
+				
+				<div class="form-box">
+					<label for="message"></label>
+					<textarea name="message" id="message" cols="20" rows="10" max="50">Whatever you want to say!</textarea>
+				</div><br/>
+				
+				<div class="form-box">
+					 <button type="submit" id="btn">Submit</button>
+				</div>
+			</div>
+		</form>
+	</main>
+	<hr/>
+	
+	<footer>
+		<p><samll>All right reserved &copy; 2021. Zaris&trade;.</samll></p>
```

Comment:
What is the samll tag for?

Reply:
@Ireti203, please I have updated the file, thanks.